ipfx/sweep.py

```diff
@@ -9,7 +9,12 @@ def __init__(self, t, v, i, clamp_mode, sampling_rate, sweep_number=None, epochs
         self.sampling_rate = sampling_rate
         self.sweep_number = sweep_number
         self.clamp_mode = clamp_mode
-        self.epochs = epochs
+
+        if epochs:
+            self.epochs = epochs
+        else:
+            self.epochs = {}
+
```

Comment:
Maybe can be a one-liner: `self.epochs = epochs if epochs else dict`

I don't remember if it should be `dict` or `{}` though...